# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/api/FileSystemManager.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/api/FileSystemManager.cpp
--- a/dom/fs/api/FileSystemManager.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/api/FileSystemManager.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -78,7 +78,7 @@
       ->Then(
           GetCurrentSerialEventTarget(), __func__,
           [self = RefPtr<FileSystemManager>(this),
-           promise](const RefPtr<OriginPrivateFileSystemChild>& child) {
+           promise](const RefPtr<FileSystemManagerChild>& child) {
             RefPtr<FileSystemActorHolder> actorHolder =
                 MakeAndAddRef<FileSystemActorHolder>(child);
             self->mRequestHandler->GetRootHandle(actorHolder, promise);